lib/kafka/consumer.rb

```diff
@@ -230,9 +230,11 @@ def each_message(min_bytes: 1, max_wait_time: 5)
     #   is ignored.
     # @param max_wait_time [Integer, Float] the maximum duration of time to wait before
     #   returning messages from the server, in seconds.
+    # @param auto_commit_offsets [Boolean] if true, it will commit offsets when necessary.
+    #   Otherwise it won't commit offsets.
     # @yieldparam batch [Kafka::FetchedBatch] a message batch fetched from Kafka.
     # @return [nil]
-    def each_batch(min_bytes: 1, max_wait_time: 5)
+    def each_batch(min_bytes: 1, max_wait_time: 5, auto_commit_offsets: true)
```

Comment:
I think the option should go in the initializer instead and be exposed by https://github.com/ale7714/ruby-kafka/blob/ab128e762d30601867c67bc0934c1bbf613bc934/lib/kafka/client.rb#L222 – that's where the current offset commit policy options are.